app/javascript/src/apps/admin/AdminDashboard.js

```
@@ -28,30 +33,59 @@ export default class AdminDashboard extends React.Component {
+      .then((result) => {
+        this.setState({
+          availableUserSpace: result.available_user_space,
+        });
+      });
+  }
+
   renderDiskInfo() {
-    const { diskAvailable, diskPercentUsed } = this.state;
-    let className = diskPercentUsed > 80 ? 'text-danger' : '';
+    const {
+      diskAvailable, diskPercentUsed, availableUserSpace
+    } = this.state;
+    const className = diskPercentUsed > 80 ? 'text-danger' : '';
 
     return (
-        <Card>
-          <Card.Body className='p-0'>
-            <InputGroup >
-              <InputGroup.Text >Disk Available (MB)</InputGroup.Text>
-              <Form.Control
-                type="text"
-                defaultValue={diskAvailable || ''}
-                readOnly
-              />
-              <InputGroup.Text >Disk Percent Used (%)</InputGroup.Text>
-              <Form.Control
-                type="text"
-                className={className}
-                defaultValue={`${diskPercentUsed}%` || ''}
-                readOnly
-              />
-            </InputGroup>
-          </Card.Body>
-        </Card>
+      <Card style={{ width: '30rem' }}>
+        <Card.Body className="p-0">
+          <InputGroup.Text>Disk Available (MB)</InputGroup.Text>
+          <Form.Control
+            type="text"
+            defaultValue={diskAvailable || ''}
+            readOnly
+          />
+          <InputGroup.Text>Disk Percent Used (%)</InputGroup.Text>
+          <Form.Control
+            type="text"
+            className={className}
+            defaultValue={`${diskPercentUsed}%` || ''}
+            readOnly
+          />
+          <InputGroup.Text>Default User Available Space (MB)</InputGroup.Text>
+          <InputGroup>
+            <Form.Control
+              type="number"
+              min="0"
+              defaultValue={availableUserSpace || ''}
+              onChange={(event) => this.setState({ availableUserSpace: event.target.value })}
```

Doesn't seem to accept floats that start with `0`?
When I enter `0.5` and hit "Save", the entry seems to fail silently. I.e., `0.5` isn't shown in "Disk Usage" column in "User Management" tab of the admin interface.

Also, I can only increase the value (e.g., from `1` to `2`). Decrements (e.g., from `2` back to `1`) don't seem to be accepted and fail silently.

---

Decrements works for me. It only decreases the default value, users will not be decremented; lower value only counts for new users. Float Values are not intended.

---

So changes in the default value are only applied for new users, not existing users, got it.
If floats aren't intended, could the input be restricted to ints?

---

Changes in the default value are applied for existing users, but only if higher than its current quota.

---

> "Could the input be restricted to ints": I don't know how.

Copilot suggests:

"To restrict the input of the `Form.Control` to integers, you can use the `step` attribute with a value of `1`. This ensures that the input only allows integer values. Here is the modified code:

```jsx
<Form.Control
  type="number"
  min="0"
  step="1" // This line ensures only integer values are allowed
  defaultValue={availableUserSpace || ''}
  onChange={(event) => this.setState({ availableUserSpace: event.target.value })}
/>
```
"